src/main/java/seedu/duke/Deadline.java

```diff
@@ -0,0 +1,57 @@
+package seedu.duke;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * Creates deadlines that are a subtype of Task and stores the name of the deadline and its deadline.
+ */
+public class Deadline extends Task {
+    private LocalDate localDate;
+    private String due;
+
+    /**
+     * Initializes an instance of Deadline.
+     *
+     * @param description Name of the task.
+     * @param due Date that the task will be due by.
+     */
+    public Deadline(String description, String due) {
+        super(description);
+        LocalDate formatDate = LocalDate.parse(due);
+        this.localDate = formatDate;
```

Comment:
Perhaps `this` is unnecessary? The basic coding standard doesn't require this, but is suggested by the [advanced style guide](https://se-education.org/guides/conventions/java/index.html#variables).